db/migrate/20160614175042_translate_outdated_sequence_identifiers.rb

```diff
@@ -0,0 +1,25 @@
+class TranslateOutdatedSequenceIdentifiers < ActiveRecord::Migration
+  def up
```

Comment:
I'd rather not since it is lengthy, it is deterministic but, due to network problems, it could cause errors on production migration.